### PR TITLE
File upload code cleanup

### PR DESCRIPTION
**Pull request by INRokh**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ const express = require("express");
 const app = express();
 const cors = require("cors");
 
+require("./config/aws");
```
Comment (INRokh) [inline app.js:5]: we forgot to add call AWS config

File: controllers/image_controller.js [hunk 2]
```diff
@@ -1,51 +1,36 @@
 const ImageModel = require("../database/models/image_model");
-
-//Upload image from React to Express
+const uuidv4 = require('uuid/v4');
 const AWS = require("aws-sdk");
 const multer = require("multer");
 const multerS3 = require("multer-s3");
-const s3 = new AWS.S3({
```
Comment (INRokh) [inline controllers/image_controller.js:7]: we use AWS config

File: controllers/image_controller.js [hunk 3]
```diff
@@ -1,51 +1,36 @@
...
-  accessKeyId: process.env.AWS_KEY_ID,
-  secretAccessKey: process.env.AWS_KEY_SECRET
-});
+const s3 = new AWS.S3();
 
-function uploadfromReact(req, res) {
```
Comment (INRokh) [inline controllers/image_controller.js:12]: // metadata stores extra information about file, we don't need it.

File: controllers/image_controller.js [hunk 4]
```diff
@@ -1,51 +1,36 @@
...
-        cb(null, { fieldName: file.fieldname });
-      },
-      key: function(req, file, cb) {
-        const ext = file.mimetype.split("/")[1];
-        cb(null, file.originalname + "_" + Date.now() + "." + ext);
-      }
-    })
-  }).array("file");
 
-  upload(req, res, function(err) {
```
Comment (INRokh) [inline controllers/image_controller.js:27]: different branches of code do the same thing

File: controllers/image_controller.js [hunk 5]
```diff
@@ -1,51 +1,36 @@
...
+    s3: s3,
+    bucket: process.env.BUCKET_NAME,
+    key: function(req, file, cb) {
+      const ext = file.mimetype.split("/")[1];
+      cb(null, uuidv4() + "." + ext);
     }
+  })
+}).array("file");
 
-    //file all good saved to s3
```
Comment (INRokh) [inline controllers/image_controller.js:34]: adding support for multiple image upload

**Review by EllieChen-Git — APPROVED** [review]
Thanks Nina! This is fantastic! The code is so much cleaner now :)

**Pull request merged**